### PR TITLE
Use and process.env.INIT_CWD to find path to app for npm5 local installs

### PR DESCRIPTION
**Pull request by PanayotCankov**
Installing from local directory, npm will set process.env.INIT_CWD to the app path, and symlink to the postinstall scripts will be resolved in __dirname

### CODE REVIEW

**Comment by sis0k0** [comment]
Currently, if you are using npm 5 and try to install a nativescript plugin with hooks from local path, the hooks it contains, won't be copied to the project directory. That's because `npm install` creates a symlink for the installed plugin (again, npm5+). As a result, if you execute a livecycle script, located inside the plugin dir (for ex. - `~/my-plugin/postinstall.js`), the value of `__dirname` will be the original location of the plugin on the drive (`~/my-plugin`), instead of `~/my-app/node_modules/my-plugin`. 
INIT_CWD is available in npm since v5.3 and it contains the path where you spawned the process from.
In short, if you run `npm install` inside `~/my-app`, INIT_CWD will be assigned to `~/my-app`.
This change fixes the mentioned problem with the hooks.
